### PR TITLE
Side-by-side standalone link support

### PR DESCRIPTION
**Pull request by zagi25**
Adds **standalone link** support to the `side-by-side` (ace1209) block.

* New `decorateStandaloneLink` helper detects links that are the sole content of their paragraph and restyles the container as a `label` with a `link-container` class, tagging the anchor as `standalone-link`.
* `decorateCardText` no longer early-returns when a card has no heading paragraph, so standalone links are decorated regardless of whether a heading is present.
* CSS adds spacing above the link container and removes the default underline from standalone links.

Resolves: [MWPW-204854](https://jira.corp.adobe.com/browse/MWPW-204854)

**Figma:** https://www.figma.com/design/6FQTDUAttFbukvtskDAkPS/Product?node-id=17317-168998&m=dev

**Test URLs:**
- Before: https://main--da-cc--adobecom.aem.page/drafts/ramuntea/firefly-w5?milolibs=site-redesign-foundation
- After: https://main--da-cc--adobecom.aem.page/drafts/ramuntea/firefly-w5?milolibs=side-by-side-link

### CODE REVIEW

**Review by overmyheadandbody — APPROVED** [review]

File: libs/mep/ace1209/side-by-side/side-by-side.js [hunk 1]
```diff
@@ -6,11 +6,25 @@ let videoObserver = null;
 
 function isSvgUrl(url) { return /\.svg(\?.*)?$/i.test(url || ''); }
 
+function decorateStandaloneLink(foreground) {
+  foreground.querySelectorAll('a').forEach((a) => {
+    const parent = a.parentElement;
+    if (!parent) return;
+    const isStandalone = parent.textContent?.trim() === a.textContent?.trim();
```
Comment (DKos95) [inline libs/mep/ace1209/side-by-side/side-by-side.js:13]: Any existing CTA button in this card — the standard authored pattern here is a link wrapped in `<em>` or `<strong>`, e.g. `<p><em><a href="...">Overlay CTA</a></em></p>` — gets silently mislabeled as a plain standalone link too, and picks up extra spacing it didn't have before.

This happens because `decorateButtons` (invoked inside `decorateBlockText`, which runs on the line right before this) already strips the `<em>`/`<strong>` wrapper for CTA buttons and leaves the paragraph containing only the bare `<a>`. By the time `decorateStandaloneLink`'s `parent.textContent === a.textContent` check runs, a CTA button's paragraph is structurally indistinguishable from a genuine standalone link's paragraph, so it matches too.

Concretely: the button's paragraph gets the new `link-container` class, which adds `margin-block-start: var(--s2a-spacing-2xs)` on top of `.foreground`'s existing flex `gap` — pushing the button down further than before on every existing card that uses a single-line CTA. That's the block's standard/default CTA pattern (confirmed via the sibling `test/blocks/side-by-side/mocks/default.html` fixture, which authors a CTA as exactly `<p><em><a href="...">Overlay CTA</a></em></p>`), so this isn't an edge case — it's a visual regression on the common path.

```suggestion
    const isStandalone = parent.textContent?.trim() === a.textContent?.trim();
    if (!isStandalone || a.classList.contains('con-button')) return;
```

**Review by DKos95 — APPROVED** [review]

**Review by NadiiaSokolova — APPROVED** [review]
Verified. Ready to be merged.
Testing details [MWPW-204854](https://jira.corp.adobe.com/browse/MWPW-204854)
